lib/coin-selection/test/spec/Cardano/CoinSelection/BalanceSpec.hs

```diff
@@ -919,7 +916,9 @@ prop_performSelection_huge_inner utxoAvailable mockConstraints (Large params) =
   where
     params' :: SelectionParams TestSelectionContext
     params' = params
-        { utxoAvailable = UTxOSelection.fromIndex utxoAvailable }
+        { SelectionParams.utxoAvailable =
+            UTxOSelection.fromIndex utxoAvailable
+        }
```

Comment:
Using `set` from `generic-lens` does not appear to work here, so I used a qualifier to remove the ambiguity.